fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -1577,4 +1590,18 @@ public void logUpdateUserPrivilege(
                 userIdentity, privilegeCollection, pluginId, pluginVersion);
         logEdit(OperationType.OP_UPDATE_USER_PRIVILEGE_V2, info);
     }
+
+    public void logUpdateRolePrivilege(
+            long roleId,
+            RolePrivilegeCollection privilegeCollection,
+            short pluginId,
+            short pluginVersion) {
+        RolePrivilegeCollectionInfo info = new RolePrivilegeCollectionInfo(
+                roleId, privilegeCollection, pluginId, pluginVersion);
+        logEdit(OperationType.OP_UPDATE_ROLE_PRIVILEGE_V2, info);
+    }
+
+    public void logDropRole(long roleId) {
```

Comment:
Best to use json KV format, even though there is only one property. To support expand properties.